simulation/traffic_simulator/src/entity/misc_object_entity.cpp

```diff
@@ -49,5 +49,15 @@ auto MiscObjectEntity::getCurrentAction() const -> const std::string
   }
   return "";
 }
+
+auto MiscObjectEntity::getDriverModel() -> const traffic_simulator_msgs::msg::DriverModel
```

Comment:
```
auto MiscObjectEntity::getDriverModel() -> const traffic_simulator_msgs::msg::DriverModel
```
↓
```
auto MiscObjectEntity::getDriverModel() const -> traffic_simulator_msgs::msg::DriverModel
```

Reply:
fixed in https://github.com/tier4/scenario_simulator_v2/pull/650/commits/d3a8c09835cca7d0b623b86317f54f9152eb257e